controllers/inventory-controller.js

```diff
@@ -3,7 +3,24 @@ const knex = require("knex")(require("../knexfile"));
 const index = async (_req, res) => {
   try {
     const data = await knex("inventories");
-    res.status(200).json(data);
+    const dataWarehouse = await knex("warehouses");
+
+    const inventoryData = data.map((item) => {
+      const container = {};
+      container.id = item.id;
+      container.warehouse_name = dataWarehouse.find(
+        (w) => w.id === item.warehouse_id
+      ).warehouse_name;
+      container.item_name = item.item_name;
+      container.description = item.description;
+      container.category = item.category;
+      container.status = item.status;
+      container.quantity = item.quantity;
+
```

Comment:
Thanks Elizabeth, this works.

Another approach is to use object destructuring to clean up things a little bit:

` const { id, warehouse_id, item_name, description, category, status, quantity } = item;
      const { warehouse_name } = dataWarehouse.find(w => w.id === warehouse_id);

      return {
        id,
        warehouse_name,
        item_name,
        description,
        category,
        status,
        quantity
      };
    })`

Reply:
Hi Ryan, thanks for pointing this out. I have pushed the changes to this PR.